EthOn.rdf

```diff
@@ -477,13 +491,36 @@
 
 
 
-    <!-- http://ethon.consensys.net/knowsOfUncle -->
+    <!-- http://ethon.consensys.net/hasUncleBeneficiary -->
+
+    <owl:ObjectProperty rdf:about="http://ethon.consensys.net/hasUncleBeneficiary">
+        <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/hasBeneficiary"/>
+        <rdfs:domain rdf:resource="http://ethon.consensys.net/Block"/>
+        <rdfs:range rdf:resource="http://ethon.consensys.net/Account"/>
+        <suggestedStringRepresentation>hasUncleBeneficiary</suggestedStringRepresentation>
+        <rdfs:comment>Relates a Block to an Account to which the reward for mining an uncle is transferred.</rdfs:comment>
+        <rdfs:label>has uncle beneficiary</rdfs:label>
+         <ns:term_status>unstable</ns:term_status>
+     </owl:ObjectProperty>
+
+
 
-    <owl:ObjectProperty rdf:about="http://ethon.consensys.net/knowsOfUncle">
+    <!-- http://ethon.consensys.net/includesUncle -->
+
+    <owl:ObjectProperty rdf:about="http://ethon.consensys.net/includesUncle">
         <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/BlockObjectProperty"/>
-        <suggestedStringRepresentation>knowsOfUncle</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Relates a Block to a known Uncle.</rdfs:comment>
-        <rdfs:label xml:lang="en">knows of Uncle</rdfs:label>
+        <rdfs:domain rdf:resource="http://ethon.consensys.net/Block"/>
+        <rdfs:range>
+            <owl:Class>
+                <owl:intersectionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://ethon.consensys.net/Block"/>
+                    <rdf:Description rdf:about="http://ethon.consensys.net/Uncle"/>
+                </owl:intersectionOf>
+            </owl:Class>
+        </rdfs:range>
+        <suggestedStringRepresentation>includesUncle</suggestedStringRepresentation>
+        <rdfs:comment xml:lang="en">Relates a Block to another Block by including it as an Uncle.  Block B can be an uncle of of Block A if it the the direct child of the k&apos;th generation ancestor of a Block B, where 2&lt;=k&lt;=7 but not a direct ancestor of A. Uncles are blockchain blocks found by a miner, when a different miner has already found another block for the corresponding place in the blockchain. They are also known as “stale blocks”.  Block A must include Block B as an Uncle.</rdfs:comment>
```

Comment:
@jo-tud My interpretation of the last sentence is 'Block A must include Block B as an uncle for this object property to be valid'. I don't think @sunnya97 meant that miners are forced to include Uncles, rather that it is possible for Block B to exist but not be included in Block A, in which case this property would not apply. But regardless, I see how it can be misleading, and it really doesn't add anything to the definition, so I will remove it.